addons/mass_mailing/static/src/js/snippets.editor.js

```diff
@@ -71,24 +80,24 @@ export const MassMailingSnippetsMenu = snippetsEditor.SnippetsMenu.extend({
                 this._activateSnippet($(srcElement));
             }
         }
-    },
+    }
     /**
      * @override
      */
-    _insertDropzone: function ($hook) {
+    _insertDropzone($hook) {
         const $hookParent = $hook.parent();
-        const $dropzone = this._super(...arguments);
+        const $dropzone = super._insertDropzone(...arguments);
         $dropzone.attr('data-editor-message', $hookParent.attr('data-editor-message'));
         $dropzone.attr('data-editor-sub-message', $hookParent.attr('data-editor-sub-message'));
         return $dropzone;
-    },
+    }
     /**
      * @override
      */
-    _updateRightPanelContent: function ({content, tab}) {
-        this._super(...arguments);
-        this.$('.o_we_customize_design_btn').toggleClass('active', tab === this.tabs.DESIGN);
-    },
+    _patchForComputeSnippetTemplates($html) {
+        this.env.switchImages(this.themeParams, $html);
```

Comment:
The env is [the OWL way to share things across components](https://github.com/odoo/owl/blob/master/doc/reference/environment.md) when not all components have to use it:
Say component tree  A => B => C, A (parent) wants to share something with C (child), then it can use `useSubEnv/useChildSubEnv` and in case it needs to be reactive, use reactive or useState and put that reactive variable in the env.

I put some methods here for that purpose, but I could also use `options` since that variable is passed from parent to child.

[I did end up adding some reactive state](https://github.com/odoo/odoo/blob/d6078dbbe8864bc911cfbc7b6feccebbe28be9cc/addons/mass_mailing/static/src/js/mass_mailing_html_field.js#L31) in there to remove the bus trigger as well.